src/views/input_home.js

```diff
@@ -28,84 +28,217 @@ class InputHome extends Component {
         <SearchBar placeholder="iOS searchbar" platform="ios" {...dummySearchBarProps} />
         <SearchBar placeholder="Android searchbar" platform="android" {...dummySearchBarProps} />
         <SearchBar placeholder="Default searchbar" {...dummySearchBarProps} />
-        <View style={[styles.headerContainer, {backgroundColor: '#616389', marginTop: 20}]}>
+        <View style={[styles.headerContainer, { backgroundColor: '#616389', marginTop: 20 }]}>
           <Icon color="white" name="input" size={62} />
           <Text style={styles.heading}>Inputs</Text>
         </View>
         <View style={styles.contentView}>
-          <View style={{ backgroundColor: "#2F343B", width: SCREEN_WIDTH, alignItems: "center" }}>
+          <View style={{ backgroundColor: '#2F343B', width: SCREEN_WIDTH, alignItems: 'center' }}>
             <Text
               style={{
                 fontSize: 30,
                 marginVertical: 10,
-                fontWeight: "300",
+                fontWeight: '300',
                 marginTop: 10,
-                color: "white"
-              }}
-            >
+                color: 'white',
+              }}>
               Login
             </Text>
             <View style={styles.overlay}>
               <View style={styles.triangleLeft} />
-              <Input containerStyle={{ borderWidth: 1, borderColor: "white", borderLeftWidth: 0, height: 50, width: SCREEN_WIDTH - 80, backgroundColor: "white" }} icon={<MaterialIcon name="email-outline" color="black" size={25} />} placeholder="Email" placeholderTextColor="black" autoCapitalize="none" autoCorrect={false} keyboardAppearance="light" keyboardType="email-address" returnKeyType="next" ref={input => (this.emailInput = input)} onSubmitEditing={() => {
+              <Input
+                inputContainerStyle={{
+                  borderWidth: 1,
+                  borderColor: 'white',
+                  borderLeftWidth: 0,
+                  height: 50,
+                  width: SCREEN_WIDTH - 80,
+                  backgroundColor: 'white',
+                }}
+                icon={<MaterialIcon name="email-outline" color="black" size={25} />}
+                placeholder="Email"
+                placeholderTextColor="black"
+                autoCapitalize="none"
+                autoCorrect={false}
+                keyboardAppearance="light"
+                keyboardType="email-address"
+                returnKeyType="next"
+                ref={input => (this.emailInput = input)}
+                onSubmitEditing={() => {
```

Comment:
this is great. Prettier makes this very readable. 👍 

Thanks for adding prettier.